chromium_src/chrome/browser/notifications/notification_platform_bridge_mac.mm

```diff
@@ -0,0 +1,79 @@
+#include "chrome/browser/notifications/notification_platform_bridge_mac.h"
```

Comment:
Copyright statements are missing.

Reply:
Fixed (this is just a revert of a previous PR as we now need both native and custom ad notifications)